controllers/be-nc-news-controller.js

```diff
@@ -1,7 +1,13 @@
 const { fetchTopics } = require('../models/be-nc-news-model');
+const endpoints = require('../endpoints.json');
 
-exports.getTopics = (req, res) => {
+exports.getTopics = (req, res, next) => {
     return fetchTopics().then((topics) => {
         res.status(200).send({ topics })
     })
+    .catch(next)
+}
+
+exports.getEndpoints = (req, res, next) => {
+    return res.status(200).send(endpoints)
```

Comment:
You could improve this by putting it in an object on a key of `endpoints`, to make it more consistent with the way you are sending other responses 🕵️

Reply:
Thanks Katherine!